index.bs

```diff
@@ -2300,6 +2302,75 @@ slots</a> described in the following table:
         1. Return <code>wMaxPacketSize</code> of |endpointDescriptor|.
 </dl>
 
+# The USB Blocklist # {#blocklist}
+
+<xmp class="idl">
+  // USBBlocklistEntry is never exposed.
+  dictionary USBBlocklistEntry {
+    required unsigned short idVendor;
+    required unsigned short idProduct;
+    required unsigned short bcdDevice;
+  };
+</xmp>
+
+This specification relies on a <a href="https://raw.githubusercontent.com/WICG/webusb/main/blocklist.txt">blocklist.txt</a>
+file in this repository to restrict the set of devices a website can access.
+
+The result of <dfn>parsing the blocklist</dfn> at a {{URL}} |url| is a [=list=]
+of {{USBBlocklistEntry}} objects produced by the following algorithm:
+
+  1. Fetch |url| and let |contents| be its body, decoded as UTF-8.
+  1. Let |lines| be the result of invoking
+     {{String/split(separator, limit)}} on |contents| with separator
+     <code>'\n'</code>.
+  1. Let |blocklist| be an empty [=list=].
+  1. [=list/For each=] |line| of |lines|:
+      1. Let |commentBegin| be the result of invoking
+         |line|.{{String/indexOf()}} with <code>'#'</code>.
+      1. If |commentBegin| is not -1, set |line| to the result of invoking
+         |line|.{{String/substring()}} with 0 and |commentBegin|.
```

Comment:
Use [collect a sequence of code points](https://infra.spec.whatwg.org/#collect-a-sequence-of-code-points) to do this.